demo/models.py

```diff
@@ -13,7 +13,17 @@
 from wagtail.wagtailsnippets.models import register_snippet
 from wagtail.wagtailsearch import index
 
-from wagtail.wagtailcore.blocks import TextBlock, ChooserBlock, StructBlock, ListBlock, StreamBlock, FieldBlock, CharBlock, RichTextBlock, PageChooserBlock, RawHTMLBlock
+from wagtail.wagtailcore.blocks import (TextBlock, 
+                                        ChooserBlock, 
+                                        StructBlock, 
+                                        ListBlock, 
+                                        StreamBlock, 
+                                        FieldBlock, 
+                                        CharBlock, 
+                                        RichTextBlock, 
+                                        PageChooserBlock, 
+                                        RawHTMLBlock)
+	
```

Comment:
While this is good, there is no need to refactor any code in the demo app - this will all be moved into a better app structure. Just for future reference so you're not wasting your time.